modules/service_container_doctrine/modules/service_container_doctrine_block/service_container_doctrine_block.info

```diff
@@ -0,0 +1,12 @@
+name = Services Container Doctrine Block
```

Comment:
Nice proof-of-concept, but will want this in a follow-up PR. This is performance critical and bean got it wrong, we need to be better, can leave in for now, but I want it at least hidden=TRUE before merging in.